src/main/scala/diplomacy/EspressoAddressDecoder.scala

```diff
@@ -0,0 +1,47 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomacy
+
+import chisel3._
+import chisel3.util.BitPat
+import chisel3.util.log2Ceil
+import chisel3.util.experimental.decode._
+
+object EspressoAddressDecoder
```

Comment:
I think it should be the name `EspressoAddressDecoder`: if user not installing `espresso`, this is gone be `QMCDecoder` from chisel

Reply:
I think you can move logic in `EspressoAddressDecoder` to `AddressDecoder`, since after this PR is merged, original `AddressDecoder` is going to be dead code.